projects/ad_fmclidar1_ebz/README.md

```diff
@@ -0,0 +1,115 @@
+
+# AD_FMCLIDAR1_EBZ HDL reference design
+
+## Overview
+
+The following design supports both Xilinx and Intel FPGA's. The [AD_FMCLIDAR1_EBZ](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/AD-FMCLIDAR1-EBZ.html)
+prototyping system connects to the FPGA carrier board through a FMC (FPGA Mezzanine Cad)
+high pin count connector.  
+
+Detailed user guide of the prototyping platform can be found [here](https://wiki.analog.com/resources/eval/user-guide/ad-fmclidar1-ebz).
```

Comment:
The correct link is: https://wiki.analog.com/resources/eval/user-guides/ad-fmclidar1-ebz